### ISSUE TITLE
Quasar cannot connect to any seeders

### ISSUE THREAD

**Issue by ipitythefool101** [scakemyer/plugin.video.quasar#472]
For the past week or so, Quasar cannot connect to a single seeder.  On a torrent with 4999 seeders, it gets stuck on "connected to 0/4999 seeds", for example.  After a couple minutes, a notice comes up that Quasar timed out, but the buffering window will stay up until closed.

Loading up the same torrent in uTorrent will connect fairly quickly.  I've tried uninstalling and reinstalling the addon but this issue persists.

Windows 10
Jarvis
Version 0.9.33

**Comment by mannnyyy** [comment]
Hi all,

Exact same problem for me (Kodi 16 & Kodi 17) on Rpi2, and for a few of my colleagues (Kodi 16, Rpi2).

A Rpi2 reboot seems to fix the problem for now. 

We can play a few movies/tvshows without any problem, and then if we want to do the same the next day, we have to reboot the system. It seems to me that it may be a problem when we come back from stand by. 

Maybe the ethernet port don't wake up properly? but, if so, it wouldn't also appear on windows 10 with different hardware... Difficult to find a reason to look for...

Thanks for all the work on this add-on.

**Comment by scakemyer** [comment]
Standby could definitely be causing this. There used to be an internet connection check, which iirc reloaded libtorrent settings and connections, but it was making DNS requests every second so it was removed for the sake of privacy and connection bloat in #64

**Comment by ipitythefool101** [comment]
Any take on my original issue?  No matter what I do, quasar is the only torrent application that cannot connect to even a single seeder.  Utorrent works, popcorn time works.  I've tried version 0.9.1 as well, and I get the exact same issue.

I am stuck on "Finding 0.00%" for torrents with over 5,000 seeders.  I have also tried uninstalling and reinstalling quasar and resetting windows firewall and readding quasar.  Nothing works.  Quasar worked fine one time, and then never again.

**Comment by kristianschneider** [comment]
I experience this too. Used to work great now nothing. uTorrent works as expected.
 Is there perhaps a way to enable more debugging so that we can assist in solving this?

**Comment by entr** [comment]
Does clearing the cache help? It helped in my situation.

**Comment by kristianschneider** [comment]
Unfortunately not.

**Comment by mannnyyy** [comment]
I resolved this problem by targeting the stand-by process.

My Sony TV was enabled for HDMI-CEC (https://en.wikipedia.org/wiki/HDMI#CEC). Using that function I could manage my Raspberry/Kodi and Playstation using my Sony TV remote controller.

Unfortunately, when I did stop my TV, I think that it was also sending a stand-by or something message to the Raspberry. So when I get back to it the next day, I had the "finding" error.

It's relatively new problem, as I was using my Raspberry and Kodi with my TV remote control whithout any problem with Kodi 14 & 15, and Openelec.

Hope there will be a fix to avoid the problem and allow us to use HDMI CEC at the same time.

Or maybe I'm totally wrong with my understanding of this issue :-)

**Comment by vitalyg149** [comment]
I found the same issue can be resolved by using a third party firewall. I was having this issue on PC and solved it by installing private firewall (link below). For some reason every time i reset windows firewall it started working again for a month or so then stop again. Same issue with pulsar...

https://www.privacyware.com/personal_firewall.html

**Comment by ipitythefool101** [comment]
It's interesting in that I've already tried clearing cache, uninstalling and reinstalling the addon, resetting windows firewall, turning OFF windows firewall, even uninstalling and reinstalling Kodi.  Nothing works.

Also, I got a brand new tablet yesterday, installed kodi, installed Quasar, and had literally the exact same issue.  Brand new machine, brand new install, ONLY running Quasar.  I'm at a complete loss as to what makes Quasar so incredibly finicky compared to any other torrent streaming programs when it comes to connecting to seeders.
